File: kraken/src/main/java/glitch/kraken/object/json/list/Clips.java
package glitch.kraken.object.json.list;

import com.google.common.collect.ImmutableList;
import com.google.gson.annotations.SerializedName;
import glitch.api.objects.json.interfaces.CursorList;
import glitch.kraken.object.json.Clip;
import lombok.Data;

@Data
public class Clips implements CursorList<Clip> {
    private final ImmutableList<Clip> data;
    @SerializedName("_cursor")
    private final String cursor;
}

File: kraken/src/main/kotlin/glitch/kraken/object/json/collections/Clips.kt
package glitch.kraken.`object`.json.collections

import com.google.gson.annotations.SerializedName
import glitch.api.objects.json.interfaces.CursorList
import glitch.kraken.`object`.json.Clip

data class Clips(
        override val data: List<Clip>,
        @SerializedName("_cursor")
        override val cursor: String
) : CursorList<Clip>
